stream/client.py

```diff
@@ -101,6 +111,10 @@ def get_full_url(self, relative_url):
         url = self.base_url + self.version + '/' + relative_url
         return url
 
+    def get_full_personal_url(self, relative_url):
+        url = self.base_url + '/' + relative_url + '/'
```

Comment:
we need a prefix in order to be able to route to the personalization infrastructure

so instead of `/` do `/personalization/` + relative_url ....